src/midi/midi_alsa.h

```diff
@@ -29,205 +29,32 @@
 #define ALSA_PCM_OLD_HW_PARAMS_API
 #define ALSA_PCM_OLD_SW_PARAMS_API
 #include <alsa/asoundlib.h>
```

Comment:
Suggest adding a TODO here to use modern 1.0.x ALSA API.

These OLD flags turn on ALSA 0.9.x API (circa 2002), which according to this recent Timidity++ issue, cause the ALSA headers to expose some symbols that cause ambiguous linking, which is a problem for  LTO builds: https://bugs.gentoo.org/763612